### ISSUE TITLE
[DEA] Subject form element should work without the need for Search option to be enabled

### ISSUE THREAD

**Issue by mahalakshme** [avniproject/avni-webapp#1342]
#### Mobile app:
If displayAllMembers checked, then Attendance form element rendered
else single or multi-select  - shows search option when more than 30 members need to show

#### DEA:
If displayAllMembers checked, then rendered, Attendance element
If nothing selected(default) - nothing shown
If search selected search shown

Individual:
until 30, then search

#### App designer:
- Remove the Search option in App designer
- Only in forms linked to Group and Household subject type show 'Display all group members'

#### DEA:
- By default for Subject concept type, Search should be shown in the DEA like available for mobile app
- If displayAllGroupMembers is enabled, then Attendance widget with checkboxes in the right need to show like available now

### Analysis notes:
- answersToShow is not supported in webapp since anyways we support only search and not select even for less than 30
answersToShow is  supported in mobile app when select UI renders with less than 30 members, but doesn't work when displayAllMembers checked
- I think Search option was added before may be because Subject form element was not working and hence they might have hesitated to support Select in that(when working for attendance form element). But now it is decided we need not support Select in Subject form element for now in DEA since the number of subjects will be more in DEA

**Comment by vinayvenu** [comment]
@mahalakshme It took about half an hour to figure this out. This could be part of our self-service "things to make simpler"

**Comment by mahalakshme** [comment]
@vinayvenu yeah it has caused confusions several times, When we supported subject concept in DEA, this issue got introduced. Before that it was not there. Will prioritise it soon.
